client/components/mma/shared/ProductDescriptionListTable.tsx

```diff
@@ -200,7 +202,10 @@ export const ProductDescriptionListTable = (
 						<dt css={tableEntryTitleCss(!!tableEntry.spanTwoCols)}>
 							{tableEntry.title}
 						</dt>
-						<dd css={tableValueCss(!!tableEntry.spanTwoCols)}>
+						<dd
+							css={tableValueCss(!!tableEntry.spanTwoCols)}
+							data-qm-masking="blocklist"
```

Comment:
just curious, what would we see here?

Reply:
<img width="838" alt="image" src="https://github.com/guardian/manage-frontend/assets/115992455/856e00fd-20af-46b1-b832-e04f316dd7f1">

It should mask out the sub id